code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -204,6 +204,30 @@
 		if("harm")
 			wuv(-1, M)
 
+/mob/living/simple_animal/pet/cat/attack_alien(mob/living/carbon/alien/humanoid/M)	//what the xeno was after all along in Alien
+	if(M.a_intent == "help" && M.caste == "h")
+		if (icon_state == "cat" || icon_state == "cat_rest" || icon_state == "cat_sit")

```

Comment:
Here you'd probably use `if icon_state in list("cat", "cat_rest", "cat_sit")` or something of the sort.